spec/spec_helper.rb

```diff
@@ -45,49 +44,49 @@
   # The settings below are suggested to provide a good initial experience
   # with RSpec, but feel free to customize to your heart's content.
 
-  # These two settings work together to allow you to limit a spec run
-  # to individual examples or groups you care about by tagging them with
-  # `:focus` metadata. When nothing is tagged with `:focus`, all examples
-  # get run.
+# These two settings work together to allow you to limit a spec run
+# to individual examples or groups you care about by tagging them with
+# `:focus` metadata. When nothing is tagged with `:focus`, all examples
+# get run.
   config.filter_run :focus
   config.run_all_when_everything_filtered = true
 
-  # Allows RSpec to persist some state between runs in order to support
-  # the `--only-failures` and `--next-failure` CLI options. We recommend
-  # you configure your source control system to ignore this file.
+# Allows RSpec to persist some state between runs in order to support
+# the `--only-failures` and `--next-failure` CLI options. We recommend
+# you configure your source control system to ignore this file.
   config.example_status_persistence_file_path = "spec/examples.txt"
 
-  # Limits the available syntax to the non-monkey patched syntax that is
-  # recommended. For more details, see:
-  #   - http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax
-  #   - http://www.teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
-  #   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#new__config_option_to_disable_rspeccore_monkey_patching
+# Limits the available syntax to the non-monkey patched syntax that is
+# recommended. For more details, see:
+#   - http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax
+#   - http://www.teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
+#   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#new__config_option_to_disable_rspeccore_monkey_patching
   config.disable_monkey_patching!
 
-  # Many RSpec users commonly either run the entire suite or an individual
-  # file, and it's useful to allow more verbose output when running an
-  # individual spec file.
+# Many RSpec users commonly either run the entire suite or an individual
+# file, and it's useful to allow more verbose output when running an
+# individual spec file.
   if config.files_to_run.one?
-    # Use the documentation formatter for detailed output,
-    # unless a formatter has already been configured
-    # (e.g. via a command-line flag).
-    config.default_formatter = "doc"
+# Use the documentation formatter for detailed output,
+# unless a formatter has already been configured
+# (e.g. via a command-line flag).
+    config.default_formatter = 'doc'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.